docs/tutorials/excel-tutorial.md

```diff
@@ -1007,3 +985,4 @@ In this tutorial, you've created an Excel task pane add-in that interacts with t
 - [Office Add-ins platform overview](../overview/office-add-ins.md)
```

Comment:
For step 6 of the final [Test the add-in](https://review.learn.microsoft.com/en-us/office/dev/add-ins/tutorials/excel-tutorial?branch=pr-en-us-3801#test-the-add-in-5) section, it would be useful to indicate which file contains the `processMessage` function.

Reply:
Did you have a different file open? Reading through, the file with this method seems to be the last one edited.

Reply:
I think I may have had a different file open because I clicked around checking on things then got to that step and wasn't sure which of the many tabs was the file I needed to update